include/materials/NEMLStress.h

```diff
@@ -24,9 +23,26 @@ class NEMLStress : public NEMLStressBase
   static InputParameters validParams();
   NEMLStress(const InputParameters & parameters);
 
-protected:
-  /// File name of the NEML XML database
-  FileName _fname;
-  /// Model name in NEML XML database
-  std::string _mname;
+private:
+  /// substitute in values for the NEML variables names in the xml file
+  std::vector<Real> constructNemlSubstitutionList(const std::vector<std::string> & nemlNames) const;
+
+  /// return string of words that string list 1 has that are not found in string list 2
```

Comment:
Follow the MOOSE documentation standards described under "Documentation" in this page: https://www.mooseframework.org/framework_development/code_standards.html